extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,14 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+[dir=rtl].amp-story-shopping-tag-inner {
```

Comment:
Please add a comment on this since it's not typical to prevent row-reverse when in RTL. 
It could be something like: "Maintains dot position in RTL mode".